docs/recipes/add_model_setup.rst

```diff
@@ -30,96 +30,190 @@ Implement a New Model
+
+                     [ ... ]
+
+                     available_versions:
+                     - 2.0-o
+                     - 2.0-esm-interface
+                     - '1.4'
+                     - '1.4-recom-mocsy-slp'
+                     - 2.0-esm-interface-yac
+                     - 2.0-paleodyn
+                     - 1.4-recom-awicm
+                     - '2.0'
+                     - '2.0-r' # OG: temporarily here
+                     choose_version:
+                       '1.4-recom-awicm':
+                         destination: fesom-1.4
+                         branch: co2_coupling
+                       '1.4-recom-mocsy-slp':
+                         branch: fesom-recom-mocsy-slp
+                         destination: fesom-1.4
+
+                     [ ... ]
+
+               .. tab:: fesom-2.0.yaml
+
+                  .. code-block:: yaml
+
+                     [ ... ]
+
+                     choose_version:
+                       '2.0':
+                         branch: 2.0.2
+                         git-repository:
+                         - https://gitlab.dkrz.de/FESOM/fesom2.git
+                         - github.com/FESOM/fesom2.git
+                         install_bins: bin/fesom.x
+                       2.0-esm-interface:
+                         branch: fesom2_using_esm-interface
+                         destination: fesom-2.0
+                         git-repository:
+                         - https://gitlab.dkrz.de/a270089/fesom-2.0_yac.git
+                         install_bins: bin/fesom.x
+
+                     [ ... ]
+
+
+   .. note:: These are just examples of model configurations, but the parser used by `ESM-Tools`
+      to read `yaml` files (`esm_parser`) allows for a lot of flexibility in their configuration;
+      i.e., imagine that the different versions of your model are in different repositories,
+      instead of in different branches, and their paths to the binaries are also different. Then
+      you can include the ``git-repository`` and ``install_bins`` variables inside the corresponding
+      version case for the ``choose_version``.
+
+9. You can now check if `esm_master` can list and install your model correctly::
 
     $ esm_master
 
    This command should return, without errors, a list of available models and versions including yours.
-   Then you can actually try installing your model in the desire folder::
+   Then you can actually try installing your model in the desired folder::
 
     $ mkdir ~/model_codes
     $ cd ~/model_codes
     $ esm_master install-your_model-version
 
-8. If everything works correctly you can check that your changes pass `flake8`, commit your changes, push
-   them to the ``origin`` and submit a pull request through GitHub (see steps 5-7 in
-   :ref:`contributing:Contribution to esm_tools Package`).
+10. If everything works correctly you can check that your changes pass ``flake8``, commit your changes, push
```

Comment:
Line 201: We could give a small example of how to use `flake8`:
`flake8 path/to/code/`

And maybe link `flake8` to a documentation https://flake8.pycqa.org/en/latest/index.html#